contrib/test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -ex
 
+set -e
```

Comment:
This is a big bug! I wonder how it ever worked till now

Reply:
I guess we were getting some false positives in the CI runs. `rust-bitcoin/contrib/test.sh` doesn't have this either :)